src/helpers/config.ts

```diff
@@ -158,14 +168,16 @@ exports.generateRedirects = async ({ netlifyConfig, basePath, i18n }) => {
   }
 }
 
-exports.getNextConfig = async function getNextConfig({ publish, failBuild = defaultFailBuild }) {
+export const getNextConfig = async function getNextConfig({ publish, failBuild = defaultFailBuild }) {
   try {
-    const { config, appDir, ignore } = await readJSON(join(publish, 'required-server-files.json'))
+    const { config, appDir, ignore }: RequiredServerFiles = await readJSON(join(publish, 'required-server-files.json'))
     if (!config) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
       return failBuild('Error loading your Next config')
     }
     return { ...config, appDir, ignore }
-  } catch (error) {
+  } catch (error: unknown) {
```

Comment:
It is best practice to have errors as `unknown` since there's no good way to predict the type of an error